Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -4284,6 +4284,12 @@ - (void)setupUserDefaults
     {
         RiotSettings.shared.showDecryptedContentInNotifications = BuildSettings.decryptNotificationsByDefault;
     }
+    
+    // Need to set `showAllRoomsInHomeSpace` to `true` for the new App Layout
+    if (BuildSettings.newAppLayoutEnaled)
+    {
+        RiotSettings.shared.showAllRoomsInHomeSpace = YES;
+    }
```

Comment:
Indeed. From a product stand point it somehow makes no sense to have an All Chats screen that displays only orphaned rooms but it's still an open point.

Reply:
Unrelated to this PR, but I think if this happens, it would be nice to see metaspaces coming to mobile like we have on Web e.g. a "Chats outside of a space" space.